integreat_cms/release_notes/current/unreleased/3858.yml

```
@@ -0,0 +1,2 @@
+en: Fix the incorrect calculation of total number of words
+de: Behebe den Fehler, dass die Gesamtanzahl an Wörter falsch berechnet wurde
```

I think the bigger issue for people was that it was saying different numbers in different places
```suggestion
en: Fix the incorrect calculation of total number of words in the translation dashboard
de: Behebe den Fehler, dass die Gesamtanzahl an Wörter in der Übersetzungsübersicht falsch berechnet wurde
```
Maybe we have a better / more correct term than *translation dashboard*…?